cpg_workflows/resources.py

```diff
@@ -337,14 +337,20 @@ def joint_calling_scatter_count(sequencing_group_count: int) -> int:
 
 
 def storage_for_joint_vcf(
-    sequencing_group_count: int | None, site_only: bool = True
-) -> int | None:
+    sequencing_group_count: int | None, site_only: bool = True, job_attrs: dict | None = None
```

Comment:
the job_attrs dict should be fine here, but we only use it for sequencing type which should always be present in `get_config()['workflow']['sequencing_type']` without the need to pass any objects around

Reply:
Good shout